src/amo/containers/CategoryPage.js

```diff
@@ -14,10 +14,21 @@ export function CategoryPageBase(props) {
 }
 
 export function mapStateToProps(state, ownProps) {
+  // HACK: This loads Firefox personas (lightweight themes) for Android
+  // until https://github.com/mozilla/addons-frontend/issues/1723#issuecomment-278793546
```

Comment:
In addition to linking here, I think it would be good to file an addons-server bug asking to insert the new mobile categories and link to that issue